docs/src/main/asciidoc/hibernate-search-orm-elasticsearch.adoc

```diff
@@ -544,38 +544,52 @@ quarkus.ssl.native=false <1>
-quarkus.hibernate-search-orm.automatic-indexing.synchronization.strategy=sync <8>
+quarkus.hibernate-search-orm.elasticsearch.version=7 <4>
+quarkus.hibernate-search-orm.elasticsearch.analysis.configurer=bean:myAnalysisConfigurer <5>
+%test.quarkus.hibernate-search-orm.automatic-indexing.synchronization.strategy=sync <6>
 
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://localhost/quarkus_test <9>
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://localhost/quarkus_test <7>
 %prod.quarkus.datasource.username=quarkus_test
 %prod.quarkus.datasource.password=quarkus_test
-%prod.hibernate-search-orm.elasticsearch.hosts=localhost:9200 <9>
+%prod.hibernate-search-orm.elasticsearch.hosts=localhost:9200 <7>
 ----
 <1> We won't use SSL so we disable it to have a more compact native executable.
 <2> Let's create a PostgreSQL datasource.
-<3> We will drop and recreate the schema every time we start the application.
-<4> We load some initial data.
-<5> We need to tell Hibernate Search about the version of Elasticsearch we will use.
+<3> We load some initial data on startup.
+<4> We need to tell Hibernate Search about the version of Elasticsearch we will use.
 It is important because there are significant differences between Elasticsearch mapping syntax depending on the version.
 Since the mapping is created at build time to reduce startup time, Hibernate Search cannot connect to the cluster to automatically detect the version.
 Note that, for OpenSearch, you need to prefix the version with `opensearch:`; see <<opensearch>>.
-<6> We point to the custom `AnalysisConfigurer` which defines the configuration of our analyzers and normalizers.
-<7> Obviously, this is not for production: we drop and recreate the index every time we start the application.
-<8> This means that we wait for the entities to be searchable before considering a write complete.
+<5> We point to the custom `AnalysisConfigurer` which defines the configuration of our analyzers and normalizers.
+<6> This means that we wait for the entities to be searchable before considering a write complete.
 On a production setup, the `write-sync` default will provide better performance.
 Using `sync` is especially important when testing as you need the entities to be searchable immediately.
-<9> For development and tests, we rely on <<dev-services,Dev Services>>,
+<7> For development and tests, we rely on <<dev-services,Dev Services>>,
 which means Quarkus will start a PostgreSQL database and Elasticsearch cluster automatically.
 In production mode, however,
 you will want to start a PostgreSQL database and Elasticsearch cluster manually,
 which is why we provide Quarkus with this connection info in the `prod` profile (`%prod.` prefix).
 
+[NOTE]
+====
+Because we rely on <<dev-services,Dev Services>>, the database and Elasticsearch schema
+will automatically be dropped and re-created on each application startup
+in tests and dev mode.
```

Comment:
Can you override it? Especially in dev mode, it might be useful. If so, maybe add something about it?

Reply:
It's just a default; I'll clarify that.